src/js_app.hpp

```diff
@@ -185,6 +201,12 @@ void AppClass<T>::constructor(ContextType ctx, ObjectType this_object, Arguments
         config.platform_version = js::Value<T>::validated_to_string(ctx, Object::get_property(ctx, result_object, platform_version_name));
         config.sdk_version = js::Value<T>::validated_to_string(ctx, Object::get_property(ctx, result_object, sdk_version_name));
     }
+    else {
+        // FIXME: When debugging RN apps, _createPlatformDescription() cannot be found
+        config.platform = "N/A";
```

Comment:
```suggestion
        config.platform = "RN debugging";
```

Reply:
For context, this was how the previous react native SDK reported its platform:

https://github.com/mongodb/stitch-js-sdk/blob/310f0bd5af80f818cdfbc3caf1ae29ffa8e9c7cf/packages/react-native/core/src/core/auth/internal/StitchAuthImpl.ts#L141-L143

Reply:
Same as [the other comment ](https://github.com/realm/realm-js/pull/3411#discussion_r529467419). I would suggest setting this to `"js-unknown"` to follow the pattern used by the legacy SDK and at the same time signal that the platform isn't known at this point. It'll most likely be react native chrome debugging, but we're checking for the existence of a function and not actually that we're running in that environment.

Reply:
keep up the good work guys. Anxiously waiting for this bug fix ! @kneth  @kraenhansen